src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/ObjectPools/PooledDisposer.NonCopyable.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.PooledObjects;
+
+[NonCopyable]
```

Comment:
💭 We could probably move this down. Not sure if there are any mutable value types in the compiler layer that need it right now.

Reply:
i'm good with moving this down.

Reply:
moved this down.